app/routes/content.js

```diff
@@ -7,6 +7,9 @@ export default Ember.Route.extend(ResetScrollMixin, {
     },
     setupController(controller, model) {
         controller.set('activeFile', model.get('primaryFile'));
+        Ember.run.scheduleOnce('afterRender', this, function() {
+            MathJax.Hub.Queue(['Typeset', MathJax.Hub, [Ember.$('.abstract')[0], Ember.$('#preprintTitle')[0]]]);  // jshint ignore:line
```

Comment:
Because `.abstract ` was already in place and specific enough, while preprintTitle didnt have anything already in place that was specific enough.